requirements.txt

```diff
@@ -0,0 +1,3 @@
+#ops==0.9.0
+https://github.com/canonical/operator/archive/master.zip
```

Comment:
Food for thoughts, should it be pinned to a specific version?

Reply:
Yeah, I'm working on that as part getting this integrated with our CI like for the metallb charms.  Part of what I was trying to do was find a tool like pipenv or pip-tools that would let me just specify the repo and have an `upgrade` script / make target that would update the pins to the current SHA of master.  Haven't quite gotten that working yet.